tests/playwright/src/specs/kind.spec.ts

```diff
@@ -54,34 +62,102 @@ beforeEach<RunnerTestContext>(async ctx => {
+    await deleteImage(page, backendImage);
+    await deleteImage(page, frontendImage);
+  } finally {
+    await pdRunner.close();
+  }
 });
 
 describe('Kind End-to-End Tests', async () => {
-  test.skipIf(skipKindInstallation)('Install Kind CLI', async () => {
-    await navigationBar.openSettings();
-    await playExpect(resourcesPage.kindResources).not.toBeVisible();
-    await statusBar.installKindCLI();
-    await playExpect(statusBar.kindInstallationButton).not.toBeVisible();
-  });
-  test('Verify that Kind CLI is installed', async () => {
-    await navigationBar.openSettings();
-    await playExpect(resourcesPage.kindResources).toBeVisible();
-  });
-  test('Kind extension lifecycle', async () => {
-    const extensionsPage = new ExtensionsPage(page);
-    await navigationBar.openExtensions();
-    const kindExtension = await extensionsPage.getInstalledExtension('Kind extension', extensionLabel);
-    await playExpect
-      .poll(async () => await extensionsPage.extensionIsInstalled(extensionLabel), { timeout: 10000 })
-      .toBeTruthy();
-    await playExpect(kindExtension.status).toHaveText('ACTIVE');
-    await kindExtension.disableExtension();
-    await navigationBar.openSettings();
-    await playExpect(resourcesPage.kindResources).not.toBeVisible();
-    await navigationBar.openExtensions();
-    await kindExtension.enableExtension();
-    await navigationBar.openSettings();
-    await playExpect(resourcesPage.kindResources).toBeVisible();
+  describe('Kind Installation', async () => {
+    test.skipIf(skipKindInstallation)('Install Kind CLI', async () => {
+      await navigationBar.openSettings();
+      await playExpect(resourcesPage.kindResources).not.toBeVisible();
+      await statusBar.installKindCLI();
+      await playExpect(statusBar.kindInstallationButton).not.toBeVisible();
+    });
+    test('Verify that Kind CLI is installed', async () => {
+      await navigationBar.openSettings();
+      await playExpect(resourcesPage.kindResources).toBeVisible();
+    });
+    test('Kind extension lifecycle', async () => {
+      const extensionsPage = new ExtensionsPage(page);
+      await navigationBar.openExtensions();
+      const kindExtension = await extensionsPage.getInstalledExtension('Kind extension', extensionLabel);
+      await playExpect
+        .poll(async () => await extensionsPage.extensionIsInstalled(extensionLabel), { timeout: 10000 })
+        .toBeTruthy();
+      await playExpect(kindExtension.status).toHaveText('ACTIVE');
+      await kindExtension.disableExtension();
+      await navigationBar.openSettings();
+      await playExpect(resourcesPage.kindResources).not.toBeVisible();
+      await navigationBar.openExtensions();
+      await kindExtension.enableExtension();
+      await navigationBar.openSettings();
+      await playExpect(resourcesPage.kindResources).toBeVisible();
+    });
   });
+  describe.skipIf(process.env.GITHUB_ACTIONS && process.env.RUNNER_OS === 'Linux')(
```

Comment:
![Create_Kind_cluster_failure](https://github.com/user-attachments/assets/a7acca67-e744-4a31-8b29-12552a21deb5)
There is an error with creating a Kind cluster using the Podman Desktop app on a Linux virtual machine. Additionally, the pods creation function does not work on the Linux VM as well.